cmd/boskos/boskos.go

```diff
@@ -68,12 +66,14 @@ var (
 	configPath = flag.String("config", "config.yaml", "Path to init resource file")
 	_          = flag.Duration("dynamic-resource-update-period", defaultDynamicResourceUpdatePeriod,
 		"Legacy flag that does nothing but is kept for compatibility reasons")
-	requestTTL              = flag.Duration("request-ttl", defaultRequestTTL, "request TTL before losing priority in the queue")
-	logLevel                = flag.String("log-level", "info", fmt.Sprintf("Log level is one of %v.", logrus.AllLevels))
-	namespace               = flag.String("namespace", corev1.NamespaceDefault, "namespace to install on")
-	port                    = flag.Int("port", 8080, "Port to serve on")
-	enableLeaderElection    = flag.Bool("enable-leader-election", false, "If leader election should be enabled. If enabled, it is possible to run Boskos with multiple replicas and only the one that is leader will receive traffic.")
```

Comment:
So did this not work previously?

Reply:
It did work but it broke upgrades when using it, as only one replica would get ready (the one that had the leader lease) and the others would not which in turn makes the deployment controller halt rollout.